src/lib/y2partitioner/widgets/overview.rb

```diff
@@ -329,24 +329,24 @@ def section_item(page, icon, children: [])
 
       # Generates a `device` tree item for given page
       #
-      # The OverviewTreePager has two kinds of items: section or device. Sections always has icon
-      # and starts expanded; devices has not icon and starts collapsed. See also {section_item}.
+      # @see #section_item
       #
-      # @param [CWM::Page] page
-      # @param [Array<CWM::PagerTreeItem>] children
+      # @param page [CWM::Page]
+      # @param children [Array<CWM::PagerTreeItem>]
       #
       # @return [CWM::PagerTreeItem]
       def device_item(page, children: [])
         CWM::PagerTreeItem.new(page, children: children, open: item_open?(page, false))
       end
 
-      # Whether the tree item for given page should be open (expanded) or close (collapsed)
+      # Whether the tree item for given page should be open or closed
```

Comment:
I didn't mean to remove the "expanded" and "collapsed" words, I was just trying to point to the wrong spelling in "closed".

Reply:
I thought that was a 2 in 1 suggestion. 

Fixed!I though that was a 2in1 suggestion. Fixed!